dockerfiles/incr.sh

```diff
@@ -55,6 +55,9 @@ while getopts “d:fhvlpt:n:u:b:r:a:” opt; do
        echo "    -u <user>          Specify the build user"
        echo "    -a <pat>           Personal access token for github packages"
        echo "    -v                 use verbose buildkit mode"
+       echo "The optional prev argument indicates a remote base image that will be pulled as the starting point."
```

Comment:
question: prev is not necessarily a remote base image ? i.e. could be used to specify a preferred local base image?